modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/streaming/AbstractBytesStreamingExtensionTestCase.java

```diff
@@ -242,7 +242,8 @@ public void resolveCursorsFromObjectParams() throws Exception {
 
     CursorStreamProvider provider = (CursorStreamProvider) flowRunner("objectToStream")
         .keepStreamsOpen()
-        .withPayload(factory.of(testEvent().getContext(), new ByteArrayInputStream(data.getBytes())))
+        .withPayload(factory.of(testEvent().getContext(), new ByteArrayInputStream(data.getBytes()),
+                                testEvent().getContext().getOriginatingLocation()))
```

Comment:
i know this is a test, but this may be confusing if looking for examples of this. Can you pass a mock or new location for this?